_episodes/013-rust-1.41.0.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "What's New in Rust 1.41"
+date: 2015-05-15T16:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e013-rust-1.41.0.mp3
+duration: "45:18"
+length: "32615549"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+Jon and Ben examine the features of [Rust 1.41](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html).
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+- [@02:39] - [Relaxed restrictions when implementing traits](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#relaxed-restrictions-when-implementing-traits)
+- [@09:54] - [`cargo install` updates packages when outdated](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#cargo-install-updates-packages-when-outdated)
+- [@12:20] - [Less conflict-prone Cargo.lock format](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#less-conflict-prone-cargolock-format)
+- [@20:27] - [More guarantees when using `Box<T>` in FFI](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#more-guarantees-when-using-boxt%3E-in-ffi)
+    - [Rust Unsafe Code Guidelines Working Group](https://github.com/rust-lang/unsafe-code-guidelines)
+- [@26:22] - `NonZero*` numeric types now implement `From<NonZero*>` for smaller integer widths
+- [@30:40] - [Reducing support for 32-bit Apple targets soon](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#reducing-support-for-32-bit-apple-targets-soon)
+- [@31:47] - Compiler frontend support for constant propagation
+    - [Inside Rust Blog - Constant propagation is now on by default](https://blog.rust-lang.org/inside-rust/2019/12/02/const-prop-on-by-default.html)
+- [@35:06] - [Cargo profile overrides](https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#overrides)
+- [@39:52] - Nested custom `Self` types
```

Comment:
A good link here is probably [https://github.com/rust-lang/rust/pull/64325/](https://github.com/rust-lang/rust/pull/64325/).